### PR TITLE
feat: use element for aria-label

### PR DESCRIPTION
**Pull request by gabriel-piltzer-trimble**
### :page_facing_up: Summary of Changes

Use Element for aria-label

### :thought_balloon: Type of Change

- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)
- [ ] Configuration change (modifies scaling or properties of existing infrastructure & services)
- [ ] Documentation change
- [ ] Other

### :clipboard: Test Plan

[comment]: # 'How do you know the changes are safe to ship to production?'
[comment]: # 'How do you know the changes will function as expected in future releases?'

### :white_check_mark: Self Code Review Checklist

PR authors and reviewers, please verify that all of these items have been completed.

- [x] My code is clean and readable
- [x] I followed standard conventions
- [ ] Component code is WCAG 2.2 compliant (if applicable)
- [ ] I have made theme files changes (if applicable)
- [ ] I have made documentation changes (if applicable)
- [ ] I have made Storybook changes including usage documentation (if applicable)
- [ ] I have tested the component thoroughly in Storybook including RTL rendering (if applicable)

### :link: Azure DevOps Work Item

[AB#573827](https://dev.azure.com/ViewpointVSO/74194628-0b6e-4b03-95a4-2d06c069dcbe/_workitems/edit/573827)

### CODE REVIEW

**Review by stuart-rimel — DISMISSED** [review]
👍

File: src/components/atoms/modus-wc-badge/modus-wc-badge.tsx [hunk 1]
```diff
@@ -54,7 +54,10 @@ export class ModusWcBadge {
 
   componentWillLoad() {
     if (!this.ariaLabel) {
```
Comment (stuart-rimel) [inline src/components/atoms/modus-wc-badge/modus-wc-badge.tsx]: don't we need to access through the element?

File: src/components/atoms/modus-wc-divider/modus-wc-divider.tsx [hunk 2]
```diff
@@ -57,7 +57,10 @@ export class ModusWcDivider {
 
   componentWillLoad() {
     if (!this.ariaLabel) {
```
Comment (stuart-rimel) [inline src/components/atoms/modus-wc-divider/modus-wc-divider.tsx]: don't we need to access through the element?

File: src/components/atoms/modus-wc-icon/modus-wc-icon.tsx [hunk 3]
```diff
@@ -45,8 +45,9 @@ export class ModusWcIcon {
   componentWillLoad() {
     if (!this.decorative && !this.ariaLabel) {
```
Comment (stuart-rimel) [inline src/components/atoms/modus-wc-icon/modus-wc-icon.tsx]: ariaLabel from the element here?

File: src/components/atoms/modus-wc-loader/modus-wc-loader.tsx [hunk 4]
```diff
@@ -58,7 +58,10 @@ export class ModusWcLoader {
 
   componentWillLoad() {
     if (!this.ariaLabel) {
```
Comment (stuart-rimel) [inline src/components/atoms/modus-wc-loader/modus-wc-loader.tsx]: don't we need to access through the element?

File: src/components/atoms/modus-wc-progress/modus-wc-progress.tsx [hunk 5]
```diff
@@ -40,8 +40,9 @@ export class ModusWcProgress {
   componentWillLoad() {
     if (!this.ariaLabel) {
```
Comment (stuart-rimel) [inline src/components/atoms/modus-wc-progress/modus-wc-progress.tsx]: don't we need to access through the element?

File: src/components/molecules/modus-wc-theme-switcher/modus-wc-theme-switcher.tsx [hunk 6]
```diff
@@ -39,8 +39,9 @@ export class ModusWcThemeSwitcher {
   componentWillLoad() {
     if (!this.ariaLabel) {
```
Comment (stuart-rimel) [inline src/components/molecules/modus-wc-theme-switcher/modus-wc-theme-switcher.tsx]: don't we need to access through the element?

**Review by stuart-rimel — COMMENTED** [review]
Looks good, just found some components that are still checking ariaLabel from the props and not the element ref. Not sure if there's a reason for that or if you'd want to just make a follow PR for those.

File: src/components/molecules/modus-wc-theme-switcher/modus-wc-theme-switcher.tsx (hunk 6 above)
Comment (gabriel-piltzer-trimble) [reply src/components/molecules/modus-wc-theme-switcher/modus-wc-theme-switcher.tsx]: Done

File: src/components/atoms/modus-wc-progress/modus-wc-progress.tsx (hunk 5 above)
Comment (gabriel-piltzer-trimble) [reply src/components/atoms/modus-wc-progress/modus-wc-progress.tsx]: Done

File: src/components/atoms/modus-wc-loader/modus-wc-loader.tsx (hunk 4 above)
Comment (gabriel-piltzer-trimble) [reply src/components/atoms/modus-wc-loader/modus-wc-loader.tsx]: Done

File: src/components/atoms/modus-wc-icon/modus-wc-icon.tsx (hunk 3 above)
Comment (gabriel-piltzer-trimble) [reply src/components/atoms/modus-wc-icon/modus-wc-icon.tsx]: Done

File: src/components/atoms/modus-wc-divider/modus-wc-divider.tsx (hunk 2 above)
Comment (gabriel-piltzer-trimble) [reply src/components/atoms/modus-wc-divider/modus-wc-divider.tsx]: Done

File: src/components/atoms/modus-wc-badge/modus-wc-badge.tsx (hunk 1 above)
Comment (gabriel-piltzer-trimble) [reply src/components/atoms/modus-wc-badge/modus-wc-badge.tsx]: Done

**Review by Kai-Richardson — APPROVED** [review]

**Review by stuart-rimel — APPROVED** [review]

**Pull request merged**